mir-ci/mir_ci/tests/robot/platforms/wayland/Screencopy.py

```diff
@@ -137,6 +144,37 @@ def _log_failed_match(self, screenshot, template):
             html=True,
         )
 
+    def _end_suite(self, data, result) -> None:
+        if not result.passed:
+            video_path = f"{self._screenshots_dir}/video.webm"
+            try:
+                subprocess.run(
```

Comment:
Any reason to not just use `ffmpeg-python` to keep this in process? Although I am fine either way

Reply:
Check what python-ffmpeg actually is ;)

In theory we could use `av`, but this is just simpler.

Reply:
Ah it just wraps a subprocess call, doesn't it... XD Okay you win. I think this is simple

Reply:
It's in the name, ffmpeg is the executable, libav is the library.